sdk/eventhub/event-hubs/test/client.spec.ts

```diff
@@ -22,7 +22,7 @@ function testFalsyValues(testFn: Function): void {
   });
 }
 
-describe("EventHubClient", function(): void {
+describe(`EventHubClient #RunnableInBrowser`, function(): void {
```

Comment:
Since we no longer use the constant in template literals, can we revert the change from " to `

Reply:
Not sure if it's a good use of time as we will anyway be revisiting these, but can do it

Reply:
We have other items of higher priority than #3474, so we won't be re-visiting them soon. It is always better to keep only the relevant changes in a PR/commit, it really helps when the net change is directly related to the issue at hand.